arangod/RestServer/LanguageCheckFeature.cpp

```diff
@@ -151,21 +152,13 @@ ErrorCode writeLanguage(arangodb::ArangodServer& server, std::string_view lang,
   return TRI_ERROR_NO_ERROR;
 }
 
-std::tuple<std::string, LanguageType> getOrSetPreviousLanguage(
-    arangodb::ArangodServer& server, std::string_view collatorLang,
-    LanguageType currLangType) {
+std::tuple<std::string, LanguageType> getPreviousLanguage(
+    arangodb::ArangodServer& server) {
   std::string prevLanguage;
   LanguageType prevType;
   arangodb::Result res = ::readLanguage(server, prevLanguage, prevType);
```

Comment:
The result of `readLanguage` is now fully ignored. If this is intentional, we should change this line to either
```suggestion
  res = ::readLanguage(server, prevLanguage, prevType);
```
or
```suggestion
  [[maybe_unused]] auto res = ::readLanguage(server, prevLanguage, prevType);
```